bin/scripts/fix_broken_ratings.js

```diff
@@ -0,0 +1,91 @@
+var pluginManager = require('../../plugins/pluginManager.js'),
+    async = require('async'),
+    crypto = require('crypto'),
+    countlyDb = pluginManager.dbConnection(),
+    countlyDrillDB = pluginManager.dbConnection("countly_drill");
+
+countlyDb.collection('apps').find({}).toArray(function(err, apps) {
+    if (!apps || err) {
+        console.log('No apps to fix');
+        countlyDb.close();
+        countlyDrillDB.close();
+        return;
+    }
+
+    function fixBrokenDeviceIDs(app, done) {
+        // find all broken app_user record containts sdk_name
+        countlyDb.collection('app_users' + app._id).find({did: /.*sdk_name=javascript_feedback_popup/}).toArray(function(err, users) {
+            if (!err) {
+                async.forEach(users, function(brokenUser, doneUser) {
+                    let fixed_device_id = brokenUser.did.substr(0, 36);
```

Comment:
Device id are not fixed length, it can be any custom string provided by a client too, so you need to strip out sdk_name... part specifically